pandas/core/missing.py

```diff
@@ -39,7 +44,9 @@
     from pandas import Index
 
 
-def mask_missing(arr: ArrayLike, values_to_mask) -> np.ndarray:
+def mask_missing(
+    arr: AnyArrayLike, values_to_mask: Union[AnyArrayLike, Scalar, Sequence[Any]]
```

Comment:
I think the `ArrayLike` for `arr` was correct here? 
(or if not, the docstring needs to be updated as well)